MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/AppearancePatch.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace MixedRealityExtension.Patching.Types
+{
+    public class AppearancePatch
+    {
+        [PatchProperty]
+        public bool? Enabled { get; set; }
+
+        [PatchProperty]
+        public Guid? MaterialId { get; set; }
+
+        public AppearancePatch() { }
```

Comment:
>       public AppearancePatch() { } [](start = 1, length = 35)

Do you need this empty constructor?

Reply:
Empty default constructor will be added during compilation.